internal/repository/mongo.go

```diff
@@ -0,0 +1,62 @@
+package repository
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/andreipimenov/golang-training-2021/internal/model"
+)
+
+type Client struct {
+	*mongo.Client
+	collection mongo.Collection
+}
+
+type mongoData struct {
+	Ticker string
+	Date   string
+	Open   string
+	High   string
+	Low    string
+	Close  string
+}
+
+type Repository interface {
+	Load(string) (model.Price, bool)
+	Store(string, model.Price)
+}
+
+func NewClient(mc *mongo.Client) *Client {
+	tmp_collection := mc.Database("db").Collection("price")
```

Comment:
Have you thought about specifying such params as DB and collection via config?

IMHO it will be a better way especially for these names ('db' and 'price')